packages/kotti-ui/source/kotti-field-number/KtFieldNumber.vue

```diff
@@ -411,7 +411,8 @@ export default defineComponent<KottiFieldNumber.PropsInternal>({
 			}
 
 			&--is-disabled {
-				color: var(--text-05);
+				color: vsar(--text-05) !important;
```

Comment:
@Juli0GT did you FT this?

- Why is `!important` needed for the color? Both, the buttons' icons and the input text already have color `var(--text-05)` when disabled. The problem for the color is mot likely in the usage.

- Adding `cursor: not-allowed;` here doesn't fix the cursor issue. It should be added to this class `kt-field-number__button--is-disabled`.

Also, @carsoli the computed `inputProps` isn't adding the `--is-disabled` class, so this isn't doing anything.

Reply:
The !important is needed because when using the field in a table it overrides the color because some scss aded into `_table.scss`:
```
table {
  	i {
		margin: 0 0.2rem;
		font-size: 1rem !important;
		color: $darkgray-300;
          }
}
```

This code from above overrides the color set in the `--is-disabled` class.

Reply:
Yes but that is the usage of `KtFieldNumber`, this component should care about its usage.
The problem is that the table is overriding the css. IMO the code in `_table.scss` should be moved into the corresponding components as scoped. 
@carsoli what do you think?

Reply:
after another internal discussion, can't be scoped but can cleaned up to remove classes that are no longer used on the user app and to properly target the icons under table-actions only

Reply:
PR cleaned.
It now contains only the fix for the cursor when hovering on +/- buttons on a disabled KtFieldNumber.